tasties_app/views.py

```diff
@@ -57,7 +57,9 @@ def register(request):
             messages.success(request, 'Account was created for ' + username)
             return redirect('login')
         else:
-            messages.error(request, form.error_messages)
+            for message_list in form.errors.values():
```

Comment:
I think that `form.errors` is a dictionary. Maybe you can do this with only one for loop?

Reply:
Yes, it actually looks better this way. Thanks.